x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/inference/action/InferenceAction.java

```diff
@@ -277,6 +319,12 @@ public ActionRequestValidationException validate() {
                 return e;
             }
 
+            if (chunk && ((taskType.equals(TaskType.SPARSE_EMBEDDING) || taskType.equals(TaskType.TEXT_EMBEDDING)) == false)) {
```

Comment:
There is the ANY taskType which is used when the task type is specified in the REST api. e.g. the `POST _inference/inference_id` route is used. ANY is acceptable in this case